draft-ietf-teas-actn-poi-applicability.md

```diff
@@ -2235,11 +2455,11 @@ or TI-LFA (MPLS based SR-TE case) through a protection port. At
 the same time MDSC, after checking that optical network connection
 is still fine, would trigger the reconfiguration of the back-up
 port of the IP router and of the muxponder to re-use the same
-WDM tunnel as the one used originally for the failed IP router port. Once
+DWDM tunnel as the one used originally for the failed IP router port. Once
 everything has been correctly configured, MDSC Global PCE could
 suggest to the operator to trigger a possible re-optimization of
 the back-up MPLS path to go back to the  MPLS primary path through
-the back-up port of the IP router and the original WDM tunnel if overall
+the back-up port of the IP router and the original DWDM tunnel if overall
```

Comment:
I-D.ietf-ccamp-wdm-tunnel-yang defines the 'WDM tunnel' model (and not the 'DWDM tunnel' model)

```suggestion
the back-up port of the IP router and the original WDM tunnel if overall
```